docs/operating-scylla/rest.rst

```diff
@@ -6,9 +6,6 @@ ScyllaDB exposes a REST API to retrieve administrative information from a node a
 administrative operations. For example, it allows you to check or update configuration, 
 retrieve cluster-level information, and more.
 
-The :doc:`nodetool </operating-scylla/nodetool>` CLI tool interacts with a *scylla-jmx* process using JMX. 
```

Comment:
I'm not familar with REST API too, but probably we shouldn't delete whole lines of description about nodetool, since native nodetool is still exists and it uses REST API.
Maybe it can be change to:

The :doc:`nodetool </operating-scylla/nodetool>` CLI tool uses the REST API to interact with the ScyllaDB process.